includes/class-wpcom-legacy-redirector.php

```diff
@@ -6,13 +6,13 @@
 class WPCOM_Legacy_Redirector {
 	const POST_TYPE   = 'vip-legacy-redirect';
 	const CACHE_GROUP = 'vip-legacy-redirect-2';
-
+	const VERSION     = '1.4.0';
```

Comment:
We've not tagged `1.4.0` yet, so this can't be correct.

Reply:
fixed to latest known stable 1.3.0 for now